clients/dotnet-core-rest/src/Tika.RestClient.IntegrationTests/Features/Factories/LocalhostRestClient.cs

```diff
@@ -9,8 +9,14 @@ public static class LocalhostRestClient
         public static IRestClient Create()
         {
             var httpClient = new HttpClient {BaseAddress = new Uri("http://localhost:3000/")};
+            var clientOptions = new ClientOptions
+            {
+                TIKA_ENABLE_MULTI_CLUSTER = false,
+                TIKA_API_ENDPOINT = "http://localhost:3000",
```

Comment:
Can we change this port to 3001? It will collide a million billion times with the backstage frontend client